content/projects/data-science-specific/data-visualisation/linux-evolution/_index.md

```diff
@@ -5,8 +5,8 @@ flavours:
 - python
 pre: '<b>MEDIUM: </b>'
 prerequisites:
-  hard:
-  - projects/tdd/simple-calculator-part1
+  soft:
```

Comment:
There may or may not be a bug in the hard/soft part of the tilde code base. Can we stick to hard prerequisites for now? I need to test that stuff a bit more